resources/service-catalog/charts/catalog/templates/apiserver-deployment.yaml

```diff
@@ -27,8 +27,9 @@ spec:
         release: "{{ .Release.Name }}"
         releaseRevision: "{{ .Release.Revision }}"
         heritage: "{{ .Release.Service }}"
-      {{ if .Values.apiserver.annotations }}
       annotations:
+        sidecar.istio.io/inject: "false"
```

Comment:
Consider moving it into values.yaml file.

Reply:
Why?

Reply:
Just to not forget about it :)
There are few cases that "inject" entries from values.yaml into spec.template.metadata.annotations using Go Template "if" blocks.
A uniform approach should be chosen: Either we ignore these blocks and add `        sidecar.istio.io/inject: "false"` or we use this mechanism so that all annotations are declared in one place in values.yaml.